### ISSUE TITLE
Saving a loaded PPTX throws exception

### ISSUE THREAD

**Issue by derkaan** [PHPOffice/PHPPresentation#210]
Hi,

I'm trying to load an existing empty presentation to add my own slides to it.
Loading etc. seems to work fine, but PHPPresentation throws an exception when I try to save it.
It says that it misses a slide in the current layout pack.
````
PHP Fatal error:  Uncaught exception 'Exception' with message 'Could not find slide layout Titelfolie in current layout pack.' in /Path/to/my/project/src/PhpPresentation/Writer/PowerPoint2007/AbstractLayoutPack.php:205
````

System: OSX
PowerPoint Template created with PowerPoint 15.20
Language: German

Any idea why this happen and/or is actually meant to copy out slides from a loaded file into a newly created?

**Comment by M-Shahbaz** [comment]
Same error:
```
$pptReader = IOFactory::createReader('PowerPoint2007');
$oPHPPresentation = $pptReader->load('pdf.pptx');

$xmlWriter = IOFactory::createWriter($oPHPPresentation, 'PowerPoint2007');
$xmlWriter->save("output153.pptx");
```


`Fatal error: Uncaught exception 'Exception' with message 'Could not find slide layout Title slide in current layout pack`

**Comment by Progi1984** [comment]
Which branch do you use ?

**Comment by derkaan** [comment]
Let me share my latest findings.
When I use a PPT-Template created with PowerPoint 2011 for Mac, then the exception will not be thrown.
If I try the same with a PowerPoint 2016 for Mac, then it doesn't work.
Please find both used files attached.

PPT-Template created with PowerPoint 2011 for Mac: [ppt-template-PP2011.pptx](https://github.com/PHPOffice/PHPPresentation/files/183759/ppt-template-PP2011.pptx)
PPT-Template created with PowerPoint 2016 for Mac: [ppt-template.pptx](https://github.com/PHPOffice/PHPPresentation/files/183760/ppt-template.pptx)

Here are some more details to reproduce the exception:

I've installed PHPPresentation via composer:
```
$cat composer.json 
{
  "require": {
    "phpoffice/phppresentation": "dev-master"
  }
}
```
Then installed it via 
````
composer install
````
This is the code that fails 

```
<?php
require_once 'vendor/autoload.php';

use PhpOffice\PhpPresentation\PhpPresentation;
use PhpOffice\PhpPresentation\IOFactory;
use PhpOffice\PhpPresentation\Style\Color;
use PhpOffice\PhpPresentation\Style\Alignment;

function createPPT($template){
    $pptLayout = new \PhpOffice\PhpPresentation\DocumentLayout();
    $pptLayout->setDocumentLayout($pptLayout::LAYOUT_SCREEN_16X10);
    $pptTMPL = \PhpOffice\PhpPresentation\IOFactory::createReader('PowerPoint2007');
		//$ppt = $pptTMPL->load("./ppt-template.pptx");
    $ppt = $pptTMPL->load("./$template");
    $ppt->setLayout($pptLayout);

    $oWriterPPTX = IOFactory::createWriter($ppt, 'PowerPoint2007');
    $oWriterPPTX->save(__DIR__ . "/sample2.pptx");

}
createPPT("ppt-template-PP2011.pptx", "PowerPoint2007");
echo "\nppt-template-PP2011.pptx processed\n==================================\n\n";
createPPT("ppt-template.pptx", "PowerPoint2007");
echo "\nppt-template.pptx processed\n============================\n\n";
```
This is the used PHP version
````
$php -v
PHP 5.6.11 (cli) (built: Jul 14 2015 16:29:05) 
Copyright (c) 1997-2015 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2015 Zend Technologies
    with the ionCube PHP Loader v4.7.5, Copyright (c) 2002-2014, by ionCube Ltd., and
    with Xdebug v2.3.3, Copyright (c) 2002-2015, by Derick Rethans
````

If it helps; this is what I find in my composer.lock
```
$cat composer.lock 
{
    "_readme": [
        "This file locks the dependencies of your project to a known state",
        "Read more about it at http://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
        "This file is @generated automatically"
    ],
    "hash": "faa9f7cc4990119283fce1199158a4bc",
    "packages": [
        {
            "name": "pclzip/pclzip",
            "version": "2.8.2",
            "source": {
                "type": "git",
                "url": "https://github.com/ivanlanin/pclzip.git",
                "reference": "19dd1de9d3f5fc4d7d70175b4c344dee329f45fd"
            },
            "dist": {
                "type": "zip",
                "url": "https://api.github.com/repos/ivanlanin/pclzip/zipball/19dd1de9d3f5fc4d7d70175b4c344dee329f45fd",
                "reference": "19dd1de9d3f5fc4d7d70175b4c344dee329f45fd",
                "shasum": ""
            },
            "type": "library",
            "autoload": {
                "classmap": [
                    "pclzip.lib.php"
                ]
            },
            "notification-url": "https://packagist.org/downloads/",
            "license": [
                "LGPL-2.1"
            ],
            "authors": [
                {
                    "name": "Vincent Blavet"
                }
            ],
            "description": "A PHP library that offers compression and extraction functions for Zip formatted archives",
            "homepage": "http://www.phpconcept.net/pclzip",
            "keywords": [
                "php",
                "zip"
            ],
            "time": "2014-06-05 11:42:24"
        },
        {
            "name": "phpoffice/common",
            "version": "0.2.5",
            "source": {
                "type": "git",
                "url": "https://github.com/PHPOffice/Common.git",
                "reference": "0525632c0becdd89a3f3895f95f549a43fae2761"
            },
            "dist": {
                "type": "zip",
                "url": "https://api.github.com/repos/PHPOffice/Common/zipball/0525632c0becdd89a3f3895f95f549a43fae2761",
                "reference": "0525632c0becdd89a3f3895f95f549a43fae2761",
                "shasum": ""
            },
            "require": {
                "pclzip/pclzip": "^2.8",
                "php": ">=5.3.0"
            },
            "require-dev": {
                "phpdocumentor/phpdocumentor": "2.*",
                "phploc/phploc": "2.*",
                "phpmd/phpmd": "2.*",
                "phpunit/phpunit": "3.7.*",
                "sebastian/phpcpd": "2.*",
                "squizlabs/php_codesniffer": "2.*"
            },
            "type": "library",
            "autoload": {
                "psr-4": {
                    "PhpOffice\\Common\\": "src/Common/"
                }
            },
            "notification-url": "https://packagist.org/downloads/",
            "license": [
                "LGPL"
            ],
            "authors": [
                {
                    "name": "Mark Baker"
                },
                {
                    "name": "Franck Lefevre",
                    "homepage": "http://rootslabs.net"
                }
            ],
            "description": "PHPOffice Common",
            "homepage": "http://phpoffice.github.io",
            "keywords": [
                "common",
                "component",
                "office",
                "php"
            ],
            "time": "2016-03-03 15:05:43"
        },
        {
            "name": "phpoffice/phppresentation",
            "version": "dev-master",
            "source": {
                "type": "git",
                "url": "https://github.com/PHPOffice/PHPPresentation.git",
                "reference": "b70f7eaddf2ccb8d44a5fec0618ac26efbf6c656"
            },
            "dist": {
                "type": "zip",
                "url": "https://api.github.com/repos/PHPOffice/PHPPresentation/zipball/b70f7eaddf2ccb8d44a5fec0618ac26efbf6c656",
                "reference": "b70f7eaddf2ccb8d44a5fec0618ac26efbf6c656",
                "shasum": ""
            },
            "require": {
                "ext-xml": "*",
                "ext-zip": "*",
                "php": ">=5.3.0",
                "phpoffice/common": "0.2.*"
            },
            "require-dev": {
                "phpdocumentor/phpdocumentor": "2.*",
                "phploc/phploc": "2.*",
                "phpmd/phpmd": "2.*",
                "phpunit/phpunit": "3.7.*",
                "sebastian/phpcpd": "2.*",
                "squizlabs/php_codesniffer": "2.*"
            },
            "suggest": {
                "ext-gd2": "Required to add images"
            },
            "type": "library",
            "autoload": {
                "psr-4": {
                    "PhpOffice\\PhpPresentation\\": "src/PhpPresentation/"
                }
            },
            "notification-url": "https://packagist.org/downloads/",
            "license": [
                "LGPL"
            ],
            "authors": [
                {
                    "name": "Mark Baker"
                },
                {
                    "name": "Ivan Lanin",
                    "homepage": "http://ivan.lanin.org"
                },
                {
                    "name": "Franck Lefevre",
                    "homepage": "http://rootslabs.net"
                }
            ],
            "description": "PHPPresentation - Read, Create and Write Presentations documents in PHP",
            "homepage": "http://phpoffice.github.io",
            "keywords": [
                "LibreOffice",
                "PowerPoint",
                "odp",
                "php",
                "ppt",
                "pptx",
                "presentations"
            ],
            "time": "2016-01-24 10:39:19"
        }
    ],
    "packages-dev": [],
    "aliases": [],
    "minimum-stability": "stable",
    "stability-flags": {
        "phpoffice/phppresentation": 20
    },
    "prefer-stable": false,
    "prefer-lowest": false,
    "platform": [],
    "platform-dev": []
}
```

**Comment by derkaan** [comment]
Any idea or update on this issue?

**Comment by camel-chased** [comment]
same here

**Comment by derkaan** [comment]
Does the master and slides names to be in english or the Powerpoint software itself configured/installed in english?

**Comment by camel-chased** [comment]
I was thinking that the language was the problem but no.
```Writer``` doesn't read what files are needed to save the file
If you read pptx over ```Reader``` you can lost every dependencies in file because
```Writer``` is not interested what may be in _rels it just assumes that everything is default
https://github.com/PHPOffice/PHPPresentation/blob/develop/src/PhpPresentation/Writer/PowerPoint2007.php#L72

I'm writing my own lib for my specific situation, because I just need to copy master and slides to separate files, im using ziparchive and read relationships from zip

**Comment by Progi1984** [comment]
@derkaan @camel-chased I linked this ticket to #161 (and a check in the roadmap #161#issuecomment-195254785).

**Comment by Progi1984** [comment]
Fixed with the PR #230
